platform/mock-radio.ts

```diff
@@ -34,10 +34,17 @@ export class MockNetwork {
   ) {
     const key = address + ':' + port.toString();
     if (this.udpListeners.has(key)) {
-      this.udpListeners[key].push(listener);
+      const listeners = this.udpListeners.get(key);
+      if (listeners !== undefined) {
+        listeners.push(listener);
+      }
+      const tlisteners = this.udpListeners.get(key);
+      if (tlisteners !== undefined) {
+        console.log(tlisteners.length);
```

Comment:
Not sure we need to log, but you could write a test to assert the length has changed.

Reply:
Left this in from debugging.  Removed.